static-checks/rpmbuild-ctest/test.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+
+import re
+from pathlib import Path
+
+from lib import util, results, versions
+
+
+def download_source(package):
+    """
+    Download src rpm and return Path of the file.
+
+    'package' is name of package to download.
+    """
+    util.subprocess_run(['yumdownloader', '--source', package], check=True)
+    package_glob = package + '*.src.rpm'
+    try:
+        src_path = next(Path.cwd().glob(package_glob))
+    except StopIteration:
+        return FileNotFoundError
```

Comment:
I don't think this is going to work. :smile:  
Try:
```python
    except StopIteration:
        raise FileNotFoundError(f".src.rpm not found for {package}")
```

Reply:
Oh god :D Right